src/importer/gooey.js

```diff
@@ -1,14 +1,54 @@
-const { scrapFrom } = require('../scraper/gdoc');
-const { launcher } = require('../utils');
+const fs = require('fs');
+const path = require('path');
+const axios = require('axios');
+const { genId, sortBy } = require('../utils');
 
-const scrap = scrapFrom('15tTzH6j6qne6WHnADVr_kHI_n2WJUD7Zk26Qdiu_SLA', {
-  name: 'gooey keys',
-  instagram: 'https://www.instagram.com/gooeykeys/',
-  discord: 'https://discord.com/invite/Wuz3dS2hkX',
-  website: 'https://www.gooeykeys.com/',
-});
+const CATALOG_JSON_URL = 'https://gooey.link/keycap-archivist.json';
+const ARTISAN_NAME = 'Gooey Keys';
+const ARTISAN_NAME_KEBAB_CASE = 'gooey-keys';
+const USE_PROVIDED_SORTING = true;
 
-launcher(scrap);
+async function scrap() {
+  try {
+    const catalog = await axios.get(CATALOG_JSON_URL).then((res) => res.data);
+
+    // Generate a unique ID for the catalog
+    catalog.id = genId(ARTISAN_NAME);
```

Comment:
if you want to keep the same catalog id use : 
```js
catalog.id = genId('gooey keys');
```

Reply:
Zekth,

Thanks so much for the reply! And with the quickness! Much appreciated.

First, that sounds great on the images point. No worries there. **Question though**: are images cached in any way that I'd need to bust (if, for instance, I changed an image for a colorway but no other data)?

As for the catalog id: in the code I've provided, the `ARTISAN_NAME` const is set to "Gooey Keys" so that line would translate to:
```js
catalog.id = genId('Gooey Keys');
```
Is this an issue? If it is, would it be possible to change my catalog on the site to "Gooey Keys" rather than "gooey keys"? Please let me know if I can facilitate in any way or if I should just change `ARTISAN_NAME` to "gooey keys".

Thanks!

Reply:
To do that just do:
```js
catalog.id = genId('gooey keys');
catalog.name = ARTISAN_NAME;
```

Regarding caching, once the images are in the bucket those are not reloaded unless the link changes.